bwrap.xml

```diff
@@ -42,7 +42,7 @@
 
 <refsect1><title>Description</title>
 <para>
-  <command>bwrap</command> is a privileged helper for container setup. You
+  <command>bwrap</command> is an unprivileged sandboxing tool. You
```

Comment:
Unfortunately, it's both: it depends how you install it. On old distributions like Debian 10 and RHEL 7, it has to be installed setuid root to work.

In some senses it's a privileged helper *and* an unprivileged sandboxing tool: bubblewrap itself is privileged, in order to make unprivileged sandboxing possible (an unfortunate collision of terminology there).

Reply:
@smcv two thoughts:
- The man page needs to only reflect the present, correct? Is there anyone left that still installs setuid? For me on Gentoo the binary is `0755`. Then maybe `an unprivileged sandboxing tool` does match present day?
- If there are still cases where it's installed setuid, how about `an unprivileged sandboxing tool (unless explicitly installed with setuid)`?

I'm happy with dropping, adjusting, sticking to the new version as you see fits.